<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowMultiScrapeDialog</class>
 <widget class="QDialog" name="TvShowMultiScrapeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>754</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TV Show Multi Scrape</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblHelp">
     <property name="text">
      <string>Please select the infos you want to be loaded. MediaElch will use the best result for each TV show and episode you selected.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabShowDetails">
        <attribute name="title">
         <string>Show Details</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="showInfosGroupBox">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Infos to load</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="MyCheckBox" name="chkTitle">
                <property name="text">
                 <string>Title</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="MyCheckBox" name="chkSeasonThumb">
                <property name="text">
                 <string>Season Thumb</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="MyCheckBox" name="chkSeasonPoster">
                <property name="text">
                 <string>Season Poster</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="MyCheckBox" name="chkPoster">
                <property name="text">
                 <string>Poster</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="MyCheckBox" name="chkNetwork">
                <property name="text">
                 <string>Network</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="MyCheckBox" name="chkCertification">
                <property name="text">
                 <string>Certification</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="MyCheckBox" name="chkRuntime">
                <property name="text">
                 <string>Runtime</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="MyCheckBox" name="chkFirstAired">
                <property name="text">
                 <string>First aired</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="MyCheckBox" name="chkRating">
                <property name="text">
                 <string>Rating</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="MyCheckBox" name="chkStatus">
                <property name="text">
                 <string>Status</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="MyCheckBox" name="chkOverview">
                <property name="text">
                 <string>Overview</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="MyCheckBox" name="chkActors">
                <property name="text">
                 <string>Actors</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="MyCheckBox" name="chkBanner">
                <property name="text">
                 <string>Banner</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="MyCheckBox" name="chkGenres">
                <property name="text">
                 <string>Genres</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="MyCheckBox" name="chkFanart">
                <property name="text">
                 <string>Fanart</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="MyCheckBox" name="chkExtraArts">
                <property name="text">
                 <string>Artwork</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="MyCheckBox" name="chkSeasonBanner">
                <property name="text">
                 <string>Season Banner</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="MyCheckBox" name="chkTags">
                <property name="text">
                 <string>Tags</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="MyCheckBox" name="chkSeasonFanart">
                <property name="text">
                 <string>Season Fanart</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="MyCheckBox" name="chkThumb">
                <property name="text">
                 <string>Thumb</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkUnCheckAll">
           <property name="text">
            <string>(Un)Check all</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEpisodeDetails">
        <attribute name="title">
         <string>Episode Details</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_22">
         <item>
          <widget class="QGroupBox" name="episodeInfosGroupBox">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lblEpisodeInfosToLoad">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Infos to load</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_22">
              <item row="3" column="1">
               <widget class="MyCheckBox" name="chkEpisodeNetwork">
                <property name="text">
                 <string>Network</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="MyCheckBox" name="chkEpisodeCertification">
                <property name="text">
                 <string>Certification</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="MyCheckBox" name="chkEpisodeTitle">
                <property name="text">
                 <string>Title</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="MyCheckBox" name="chkEpisodeFirstAired">
                <property name="text">
                 <string>First aired</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="MyCheckBox" name="chkEpisodeDirector">
                <property name="text">
                 <string>Director</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="MyCheckBox" name="chkEpisodeWriter">
                <property name="text">
                 <string>Writer</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="MyCheckBox" name="chkEpisodeOverview">
                <property name="text">
                 <string>Overview</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="MyCheckBox" name="chkEpisodeThumbnail">
                <property name="text">
                 <string>Thumbnail</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">show</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="MyCheckBox" name="chkEpisodeTags">
                <property name="text">
                 <string>Tags</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="MyCheckBox" name="chkEpisodeRating">
                <property name="text">
                 <string>Rating</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="MyCheckBox" name="chkEpisodeActors">
                <property name="text">
                 <string>Actors</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="type" stdset="0">
                 <string notr="true">both</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkEpisodeUnCheckAll">
           <property name="text">
            <string>(Un)Check all</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="h_seasonTypeLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblScraper">
         <property name="text">
          <string>Scraper</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboScraper"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblLanguage">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="LanguageCombo" name="comboLanguage"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblSeasonOrder">
         <property name="text">
          <string>Order for seasons</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboSeasonOrder">
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="chkAutoSave">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="chkOnlyId">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblOnlyId">
         <property name="text">
          <string>Update only TV shows/episodes
which have an ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblAutoSave">
         <property name="text">
          <string>Automatically save each TV show/
episode after scraping</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacerComboRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="itemCounter">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>1/20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title">
       <property name="text">
        <string notr="true">Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressItem">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressAll">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtScraperLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartScraping">
       <property name="text">
        <string>Start Scraping</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">ui/small_widgets/MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>LanguageCombo</class>
   <extends>QComboBox</extends>
   <header location="global">ui/small_widgets/LanguageCombo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>TvShowMultiScrapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>TvShowMultiScrapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>514</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
